library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -603,6 +614,8 @@ private void init(Context context, @Nullable AttributeSet attrs) {
         }
         TypedArray attr = context.obtainStyledAttributes(attrs, R.styleable.SpeedDialView, 0, 0);
         try {
+            setUseReverseAnimationOnClose(attr.getBoolean(R.styleable.SpeedDialView_sdReverseAnimationOnClose, false));
```

Comment:
Please use the `getUseReverseAnimationOnClose()` method to get the default value, this way we can define the default value only [in one place](https://github.com/leinardi/FloatingActionButtonSpeedDial/pull/48/files#diff-677471938743387ceefac73f9a1f2ceaR916).

- [ ] use `getUseReverseAnimationOnClose()`